<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterJoueurInterfaceClass</class>
 <widget class="QDialog" name="ajouterJoueurInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ajouterJoueurInterface</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Nom">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Prenom">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Position">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Arabic</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>                  Ajout d'un joueur</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom du joueur                                  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prénom du joueur                            : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date de naissance                             : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Position du joueur                            :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_valide">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>3</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Valider le joueur ajoutè</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numéro du téléphone  du joueur    : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Telephone">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_valide</sender>
   <signal>clicked()</signal>
   <receiver>ajouterJoueurInterfaceClass</receiver>
   <slot>validerEnregistrement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerEnregistrement()</slot>
 </slots>
</ui>
